inference-engine/ie_bridges/java/org/intel/openvino/Blob.java

```diff
@@ -35,6 +39,8 @@ public int size() {
 
     private static native long BlobFloat(long tensorDesc, float[] data);
 
+    private static native long wrapMatToBlob(long tensorDesc, long addr);
```

Comment:
There is no Mat in general - maybe BlobArray? (similar to BlobFloat and BlobByte).